src/scalar.rs

```diff
@@ -6,6 +6,8 @@ use std::{
     ops::{Add, AddAssign, Div, DivAssign, Mul, MulAssign, Neg, Rem, RemAssign, Sub, SubAssign},
 };
 
+use crate::{op};
```

Comment:
getting a unused import warning

Reply:
Ah. I could've sworn it was yelling at me because it couldn't see the op_ macros without this, but yeah, looks like that wasn't true.

Reply:
Elsewhere too. I'll get rid of those